tests/android/MASVS-STORAGE/MASTG-TEST-0012.md

```diff
@@ -7,6 +7,9 @@ platform: android
 title: Testing the Device-Access-Security Policy
```

Comment:
Shouldn't we also be deprecating https://mas.owasp.org/MASTG/tests/ios/MASVS-AUTH/MASTG-TEST-0064/ in this PR?

https://github.com/OWASP/owasp-mastg/issues/2950

Reply:
added a deprecation note